static/js/vue-cdr-access/tests/unit/filterTags.spec.js

```diff
@@ -1,25 +1,33 @@
-import { createLocalVue, shallowMount } from '@vue/test-utils'
-import VueRouter from 'vue-router';
+import { shallowMount, flushPromises } from '@vue/test-utils'
+import { createRouter, createWebHistory } from 'vue-router';
 import filterTags from '@/components/filterTags.vue';
+import displayWrapper from "@/components/displayWrapper";
+import searchWrapper from '@/components/searchWrapper';
 
-const localVue = createLocalVue();
-localVue.use(VueRouter);
-const router = new VueRouter({
-    routes: [
-        {
-            path: '/search/:uuid?',
-            name: 'searchRecords'
-        }
-    ]
-});
-let wrapper, facet_tags;
+let wrapper, facet_tags, router;
 
 describe('filterTags.vue', () => {
     beforeEach(() => {
+        router = createRouter({
+            history: createWebHistory(process.env.BASE_URL),
+            routes: [
+                {
+                    path: '/search/:uuid?',
+                    name: 'searchRecords',
+                    component: searchWrapper
+                },
+                {
+                    path: '/record/:uuid',
```

Comment:
Just for my own learning, was the `record` route added into all these tests because of some new requirement?

Reply:
The tests still passed, but it was giving a test warning if it wasn't in the router.